README.md

```diff
@@ -1,2 +1,79 @@
-# kntox-junit5
-Testing Knot.x with JUnit 5
+# Knot.x JUnit 5
+JUnit 5 extensions and data type converters for Knot.x tests
+
+Provides following helpers:
+
+###KnotxExtension
```

Comment:
Please add space between `###` and title.
In that way markdown formatting will not apply (you may check it by using `View` button in the PR).